src/ledger/fuzz.zig

```diff
@@ -0,0 +1,466 @@
+const std = @import("std");
+const sig = @import("../sig.zig");
+const build_options = @import("build-options");
+const ledger = @import("lib.zig");
+
+const ColumnFamily = sig.ledger.database.ColumnFamily;
+
+const allocator = std.heap.c_allocator;
+
+const Data = struct {
+    value: []const u8,
+};
+
+const cf1 = ColumnFamily{
+    .name = "data",
+    .Key = u64,
+    .Value = Data,
+};
+
+var dataMap = std.AutoHashMap(u32, Data).init(allocator);
```

Comment:
might want to add a doc-comment above this explaning that this is a simpler blockstore which we will use to make sure our data is up to date -- also since its initialized it should be `data_map`

Reply:
[e29fef7](https://github.com/Syndica/sig/pull/452/commits/e29fef7149e4106793db48735e2411d147b9fb94)